src/main/java/core/basesyntax/Truck.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+public class Truck extends Machine {
+    @Override
+    public void doWork() {
+        System.out.println("Truck started working");
```

Comment:
The message 'Truck started working' could be more informative. According to the checklist, the message should clearly indicate that it is a Truck that is working. Consider specifying the model or the type of work the Truck is doing to make the message more informative.